webpack/assets/javascripts/react_app/components/HostDetails/Audits/index.js

```diff
@@ -20,9 +19,8 @@ import {
 } from '@patternfly/react-core';
 
 import { foremanUrl } from '../../../../foreman_tools';
-import { get } from '../../../redux/API';
-import { selectAPIResponse } from '../../../redux/API/APISelectors';
 import { translate as __ } from '../../../common/I18n';
+import { useAPI } from '../../../common/hooks/API/APIHooks';
 
 const AuditCard = ({ hostName }) => {
   const [ativeAccordion, setActiveAccordion] = useState(0);
```

Comment:
Unrelated but noticed there is a typo here (and everywhere this state is used):
```suggestion
  const [activeAccordion, setActiveAccordion] = useState(0);
```